lightkurve/targetpixelfile.py

```diff
@@ -833,7 +826,7 @@ def prf_photometry(self, cadences=None, parallel=True, **kwargs):
         return prfphot
 
     def prf_lightcurve(self, **kwargs):
```

Comment:
What is a `prf_lightcurve`? It feels like we only need `extract_prf_photometry`? There's no equivalent `sap_lightcurve`.

Reply:
I am not going to rename the PRF methods for now and move them to a separate package instead.